lib/fog/libvirt/requests/compute/list_volumes.rb

```diff
@@ -69,23 +69,31 @@ def get_volume filter = { }, raw = false
               return raw ? vol : volume_to_attributes(vol)
             end
           end
-          { }
+
+          nil
```

Comment:
The rest seems reasonable, but I'm not sure about this particular change. It doesn't seem related to the other changes, and I fear it could break existing code/usage. Am I missing something and/or could we omit this part? Thanks!

Reply:
@geemus isn't this method private to the class?

The change here is to avoid adding empty hashes to the list_volumes response above. I guess the question would be if code would be relying on whether list_volumes always returns a list of empty hashes?

Reply:
Looking again, I think this should probably be ok. Especially as it is private, I think maybe I overlooked that fact when thinking about it before.

Reply:
You know what, I should have added a test, it turns out that while this avoids adding a list of hashes, the code above doesn't skip appending a nil entry

Reply:
And on further examination, that's probably alright because volume_to_attributes will return nil entries, so at least anyone's code at this point need only handle skipping nil entries.